src/base/dom/pan-and-zoom.ts

```diff
@@ -35,6 +37,19 @@ export class PanAndZoom {
         public max_zoom = 10,
         public bounds?: BBox,
     ) {
+        Preferences.INSTANCE.addEventListener(
+            "kicanvas:preferences:change",
+            () => {
+                //this.updateSettings(Preferences.INSTANCE);
+                console.log(
```

Comment:
Leftover console.log?

Reply:
I removed it because it was just verification code and was not necessary.